src/presentation/fastify-server.ts

```diff
@@ -127,6 +130,15 @@ export class FastifyServer implements ProxyServer {
     if (this.isEventVideoMixerConfigurationUpdatedEvent(parsedData)) {
       this.deviceEmitter.emitVideoMixerConfigurationUpdated(parsedData.videoMixer)
     }
+
+    if (this.isRundownEvent(parsedData)) {
+      if (parsedData.type === RundownEventType.ACTIVATED || parsedData.type === RundownEventType.REHEARSE) {
+        this.rundownEmitter.emitActiveRundownId(parsedData.rundownId)
+      }
+      if (parsedData.type === RundownEventType.DEACTIVATED) {
+        this.rundownEmitter.emitActiveRundownId(undefined)
+      }
```

Comment:
```suggestion
      if (parsedData.type === RundownEventType.DEACTIVATED) {
        this.rundownEmitter.emitActiveRundownId(undefined)
        return
      }
      this.rundownEmitter.emitActiveRundownId(parsedData.rundownId)
```

Or perhaps this one :)

```suggestion
      parsedData.type === RundownEventType.DEACTIVATED) ? this.rundownEmitter.emitActiveRundownId(undefined) : this.rundownEmitter.emitActiveRundownId(parsedData.rundownId)
```

Reply:
Since it is only the `type` that is checked then a switch could also be used :)

Reply:
To use the switch we would need to make some more drastic changes to the overall method, else we would have a mixture of if-statements and a switch in the same method that all check on the type.